sdk/cmake/set_wine_module.cmake

```diff
@@ -0,0 +1,7 @@
+
+function(set_wine_module TARGET)
+    target_include_directories(${TARGET} SYSTEM BEFORE PRIVATE
```

Comment:
`SYSTEM` looks unexpected/contradictory:

https://cmake.org/cmake/help/latest/command/target_include_directories.html
`If SYSTEM is specified, the compiler will be told the directories are meant as system include directories on some platforms. This may have effects such as suppressing warnings or skipping the contained headers in dependency calculations (see compiler documentation). Additionally, system include directories are searched after normal include directories regardless of the order specified.`

Reply:
```suggestion
    target_include_directories(${TARGET} BEFORE PRIVATE
```